index.js

```diff
@@ -26,7 +26,7 @@ module.exports = class PassportTrailpack extends Trailpack {
       return Promise.reject(new Error('No strategies found at config.passport.strategies !'))
     }
 
-    if (strategies.jwt && _.get(this.app, 'config.passport.jwt.tokenOptions.secret') === 'mysupersecuretoken') {
+    if (strategies.jwt && _.get(this.app, 'config.passport.jwt.tokenOptions.secret') === 'mysupersecuretoken') {  //eslint-disable-line
```

Comment:
No problem :) just want to avoid to many disabled lines around the project, if warnings are a problem just put on multiline. 
Thanks !